### ISSUE TITLE
Meeting 2020-07-29

### ISSUE THREAD

**Issue by dawnlwoodard** [bpbond/bendawn#14]
Well since I last saw you I've been mostly prepping for the Bob V talk, working on the abstract, working on GCAM stuff for Kate, and then on vacation.

Points from paper feedback:
- rationale for detritus value of 2%: I don't have one right now. Need to get from CMIP5 or 6 results which I haven't done. Is that a thing that needs to happen before I send to co-authors or after?
- code and data availability: is this just my github for now until we merge into main Hector?

AGU session options:
- none suggested by Mohamad for SFA work seem to really fit with this
- others: [Vulnerability of Permafrost Carbon to Climate Change](https://agu.confex.com/agu/fm20/prelim.cgi/Session/105082),  [High-Latitude Earth Systems - Their Local Responses to and Impacts on Global Climate Change](https://agu.confex.com/agu/fm20/prelim.cgi/Session/101041) 

Other questions:
- Should I follow up with Renu Joseph about the potential for collaboration?
- Conferences: I also got an email about IAMC -> does it make sense to do both that and AGU?
